hardhat.config.ts

```diff
@@ -165,6 +165,12 @@ const config: HardhatUserConfig = {
 
   scenario: {
     bases: [
+      {
```

Comment:
this would change, too, if #498 gets merged.

it would be:

```
{
  name: 'mainnet',
  network: 'mainnet',
  deployment: 'usdc',
  allocation: 0.1
}
```